src/quo2/components/lowest_price.cljs

```diff
@@ -0,0 +1,74 @@
+(ns quo2.components.lowest-price
```

Comment:
two suggestion to this file, when we have lots of styles when it hard to read code , better to create styles.cljs file and move all styles there , also please format styles maps

Reply:
Refactored